Makefile

```diff
@@ -129,20 +129,37 @@ upgrade: pre-requirements  ## update the pip requirements files to use the lates
 check-types: ## run static type-checking tests
 	mypy
 
-# These make targets currently only build LMS images.
 docker_build:
-	docker build . -f Dockerfile --target lms -t openedx/edx-platform
-	docker build . -f Dockerfile --target lms-newrelic -t openedx/edx-platform:latest-newrelic
+	docker build . -f Dockerfile --target lms     -t openedx/lms
+	docker build . -f Dockerfile --target lms-dev -t openedx/lms-dev
+	docker build . -f Dockerfile --target cms     -t openedx/cms
+	docker build . -f Dockerfile --target cms-dev -t openedx/cms-dev
 
 docker_tag: docker_build
-	docker tag openedx/edx-platform openedx/edx-platform:${GITHUB_SHA}
-	docker tag openedx/edx-platform:latest-newrelic openedx/edx-platform:${GITHUB_SHA}-newrelic
+	docker tag openedx/lms     openedx/lms:${GITHUB_SHA}
+	docker tag openedx/lms-dev openedx/lms-dev:${GITHUB_SHA}
+	docker tag openedx/cms     openedx/cms:${GITHUB_SHA}
+	docker tag openedx/cms-dev openedx/cms-dev:${GITHUB_SHA}
 
 docker_auth:
 	echo "$$DOCKERHUB_PASSWORD" | docker login -u "$$DOCKERHUB_USERNAME" --password-stdin
 
 docker_push: docker_tag docker_auth ## push to docker hub
-	docker push 'openedx/edx-platform:latest'
-	docker push "openedx/edx-platform:${GITHUB_SHA}"
-	docker push 'openedx/edx-platform:latest-newrelic'
-	docker push "openedx/edx-platform:${GITHUB_SHA}-newrelic"
+	docker push "openedx/lms:latest"
+	docker push "openedx/lms:${GITHUB_SHA}"
+	docker push "openedx/lms-dev:latest"
+	docker push "openedx/lms-dev:${GITHUB_SHA}"
+	docker push "openedx/cms:latest"
+	docker push "openedx/cms:${GITHUB_SHA}"
+	docker push "openedx/cms-dev:latest"
+	docker push "openedx/cms-dev:${GITHUB_SHA}"
+
+docker_push_kyle:
```

Comment:
Just making a note to remind you to delete this once it's no longer needed.

Reply:
Will do. Thanks!